recipe/meta.yaml

```diff
@@ -12,8 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+  script: DISABLE_NUMCODECS_AVX2= python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Do I need to break this out into separate build.bat and build.sh to get the environment variable set properly?